numpy/core/src/_simd/_simd_inc_vector.h

```diff
@@ -0,0 +1,206 @@
+#ifndef NPY_SIMD
+    #error "Not a standalone header"
+#endif
+
+/************************************
+ ** Private Definitions
+ ************************************/
+// PySequenceMethods
+static Py_ssize_t
+simd__vector_length(simd_vector *self)
+{
+    return simd_data_getinfo(self->type)->nlanes;
+}
+static PyObject *
+simd__vector_item(simd_vector *self, Py_ssize_t i)
+{
+    const simd_data_info *info = simd_data_getinfo(self->type);
+    int nlanes = info->nlanes;
+    if (i >= nlanes) {
+        PyErr_SetString(PyExc_IndexError, "list index out of range");
+        return NULL;
+    }
+    simd_data d;
+    d.qu8 = self->data + i * info->lane_size;
+    d.u64 = *d.qu64;
+    return simd_data2obj(d, simd_data_cvt2scalar(self->type));
+}
+static int
+simd__vector_contains(PyObject *self, PyObject *el)
+{
+    int ret = 0;
+    PyObject *obj = PySequence_Fast(self, "invalid argument, expected a vector");
+    if (obj != NULL) {
+        ret = PySequence_Contains(obj, el);
+        Py_DECREF(obj);
+    }
+    return ret;
+}
```

Comment:
This doesn't handle `obj == NULL` correctly, and returns `0` when it should return `-1`.
```suggestion
static int
simd__vector_contains(PyObject *self, PyObject *el)
{
    int ret = 0;
    PyObject *obj = PySequence_Fast(self, "invalid argument, expected a vector");
    if (obj == NULL) {
        return NULL;
    }
    int ret = PySequence_Contains(obj, el);
    Py_DECREF(obj);
    return ret;
}
```
Are you sure you need this function at all though? If `seq_contains` is missing, python automatically uses `seq_item` and `seq_length` for you.

Reply:
> Are you sure you need this function at all though?

It was a temporary implementation and there was an intention to optimize it.

>  If seq_contains is missing, python automatically uses seq_item and seq_length for you.

I removed it as you suggested, so I can put my focus on other sides.